<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Lokesh-PC" failures="3" tests="8" name="Test" time="68.09" errors="0" timestamp="2022-06-08T15:09:40 IST">
  <testcase classname="scenarios.api.EntityType" name="filename" time="0.013"/>
  <testcase classname="extend.api.CreateReport" name="extend" time="0.336"/>
  <testcase classname="testsuite2.test.EntityType_PUT_with_indistinct" name="Update_EntityType" time="25.945">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 400 Bad Request]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 400 Bad Request]
at auth.api.VerifyStatusLine.ensureLine200(VerifyStatusLine.java:22)
at depend_1.api.FrameSuite.EntityType_PUT_Update(FrameSuite.java:4570)
at main.api.ExecuteAPILead.Put(ExecuteAPILead.java:181)
at apache.api.Shuffle.globalAdmin(Shuffle.java:70)
at apache.api.Operation.worksheet(Operation.java:13)
at testsuite2.test.EntityType_PUT_with_indistinct.Update_EntityType(EntityType_PUT_with_indistinct.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Update_EntityType -->
  <testcase classname="testsuite2.test.EntityType_POST_with_indistinct" name="Create_EntityType" time="14.485"/>
  <testcase classname="testsuite2.test.EntityType_GET_with_indistinct" name="View_EntityType" time="5.812"/>
  <testcase classname="scenarios.api.EntityType_EntityTypeID" name="filename" time="0.001"/>
  <testcase classname="testsuite2.test.EntityType_DELETE_with_distinct_EntityTypeID" name="Delete_Specific_EntityType_using_EntityTypeID" time="16.146">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 404 Not Found]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 404 Not Found]
at auth.api.VerifyStatusLine.ensureLine200(VerifyStatusLine.java:22)
at depend_1.api.FrameSuite.EntityType_DELETE_Delete(FrameSuite.java:4712)
at main.api.ExecuteAPILead.Delete(ExecuteAPILead.java:235)
at apache.api.Shuffle.globalAdmin(Shuffle.java:73)
at apache.api.Operation.worksheet(Operation.java:13)
at testsuite2.test.EntityType_DELETE_with_distinct_EntityTypeID.Delete_Specific_EntityType_using_EntityTypeID(EntityType_DELETE_with_distinct_EntityTypeID.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Delete_Specific_EntityType_using_EntityTypeID -->
  <testcase classname="testsuite2.test.EntityType_GET_with_distinct_EntityTypeID" name="View_Specific_EntityType_using_EntityTypeID" time="5.279">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 400 Bad Request]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 400 Bad Request]
at auth.api.VerifyStatusLine.ensureLine200(VerifyStatusLine.java:22)
at depend_1.api.FrameSuite.EntityType_GET_View_Specific_EntityType(FrameSuite.java:4606)
at main.api.ExecuteAPILead.View_ID(ExecuteAPILead.java:301)
at apache.api.Shuffle.globalAdmin(Shuffle.java:67)
at apache.api.Operation.worksheet(Operation.java:13)
at testsuite2.test.EntityType_GET_with_distinct_EntityTypeID.View_Specific_EntityType_using_EntityTypeID(EntityType_GET_with_distinct_EntityTypeID.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- View_Specific_EntityType_using_EntityTypeID -->
</testsuite> <!-- Test -->
